enhancements/olm/report-sdk-operators.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Enable tracking of SDK operators in OLM metrics
+authors:
+  - "@varshaprasad96"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-05-28
+status: implementable
+---
+
+# Enable tracking of SDK operators in OLM metrics
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+This enhancement proposes the addition of operator specific labels to existing OLM metrics, in order to collect data on operators which utilize SDK to build image bundle.
```

Comment:
Can you specify that this is Red Hat operators only, as we can only send these metrics for red hat OLM operators for now.